packages/nimble-components/src/wafer-map/modules/zoom-handler.ts

```diff
@@ -6,33 +6,41 @@ import {
     ZoomTransform,
     zoomTransform
 } from 'd3-zoom';
-import type { DataManager } from './data-manager';
-import type { RenderingModule } from './rendering';
+import type { Dimensions, ZoomEvent, ZoomHandlerData } from '../types';
 
 /**
  * ZoomHandler deals with user interactions and events like zooming
  */
 export class ZoomHandler {
+    public onZoom: ((event: ZoomEvent) => void) | undefined;
```

Comment:
It would be helpful to document the pattern we're using here and describe what this does. Something like this?
```suggestion
    /*
     * This "event" is triggered when the user zooms the wafer map.
     * For simplicity this is just a callback function so it can only have one subscriber right now.
     */
    public onZoom: ((event: ZoomEvent) => void) | undefined;
```

Reply:
applied